app/shared/languagesList.ts

```diff
@@ -88,7 +88,7 @@ const availableLanguages: LanguageSchema[] = [
     rtl: true,
     ISO639_3: 'ara',
     localized_label: 'العربية',
-    translationAvailable: true,
+    translationAvailable: false,
```

Comment:
I think it is fine if we remove the property from all the items, although it is not a big difference in size, still could be good given this file is part of the bundle, what do you think?

Reply:
We talked about it and decided to keep it to be consistent and always have the property, the mid term idea with this is that is always requested to the server, same as templates for instance instead of having it as part of the bundle.